README.md

```diff
@@ -1,2 +1,69 @@
-# to-do-using-react-js
+# To-Do App Using React js
```

Comment:
Here in your README.md file, you're missing some important information, like how to set up the repo.

[OPTIONAL] You can add all the other sections given at [this template](https://github.com/microverseinc/readme-template) for a professional README file

Reply:
Dear, I have incorporated all the changes in the repository. You are requested to review the code again, please.